bip-musig2.mediawiki

```diff
@@ -514,6 +524,10 @@ The reference implementation is for demonstration purposes only and not to be us
 === Modifications to Nonce Generation ===
 
 Implementers must avoid modifying the ''NonceGen'' algorithm without being fully aware of the implications.
+
+One potential pitfall emerges when ''NonceGen'' is altered such that ''k<sub>1</sub>'' and ''k<sub>2</sub>'' are drawn uniformly at random instead of setting them to the output of a ''hash'' function that includes ''pk'' as input.
+The result is that in some situations, an adversary can forge a signature by influencing the individual tweak of a victim signer after seeing the ''pubnonce'' (see [https://github.com/jonasnick/musig2-tweaking here] for the details).
```

Comment:
Hm I'm not a fan of footnotes. The partial signature forgery footnote is different because it explains the attack in detail and it  wouldn't fit the surrounding text.